scripts/cf-cdn.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+CDN_URL="flybywirecdn.com"
+CDN_PURGE_LINK="https://flybywirecdn.com/purgeCache?url=http://flybywirecdn.com"
+CDN_DIR=${1:-"addons/a32nx/test-3"}
+LOCAL_DIR=${2:-"./build-modules"}
+
+MAX_RETRY=5
+
+upload () {
+    DEST="$CDN_URL/$CDN_DIR/$(basename -- "$1")"
+    echo "Syncing file: $1"
+    echo "Destination: $DEST"
+
+    # Try to upload the file up to MAX_RETRY times before failing
+    counter=0
+    until curl --fail -X PUT -H "X-FBW-Access-Key: $CLOUDFLARE_BUCKET_PASSWORD" --data-binary -T "$1" "$DEST"
```

Comment:
Afaik you should either use --data-binary or -T, but not both

Reply:
Oops that wasn't intentional. Will fix

Reply:
Done.,